hs_core/forms.py

```diff
@@ -498,11 +498,13 @@ def __init__(self, *args, **kwargs):
 
 class PartyForm(ModelForm):
     def __init__(self, *args, **kwargs):
+        if 'initial' in kwargs:
+            if kwargs['initial']['description']:
```

Comment:
Is there a possibility of getting KeyError if the Party model has the 'description' attribute?